egs/wsj/s5/utils/combine_ali_dirs.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Copyright 2016  Xiaohui Zhang  Apache 2.0.
+
+# This srcipt operates on alignment directories, such as exp/tri4a_ali
+
+# Begin configuration section. 
+cmd=run.pl
+extra_files= 
+num_jobs=4
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+if [ -f path.sh ]; then . ./path.sh; fi
+. parse_options.sh || exit 1;
+
+if [[ $# -lt 3 ]]; then
+  echo "Usage: $0 [options] <ref-data-dir> <dest-ali-dir> <src-ali-dir1> <src-ali-dir2> ..."
```

Comment:
Just call this <data> and in the code call it data, not ref_data_dir.  This is the normal pattern; in some sense, it's *always* for reference.